### PR TITLE
LPS-205450 Add searchCategoryId parameter to the portletURL when it i…

### PR DESCRIPTION
**Pull request by SamZiemer**
https://liferay.atlassian.net/browse/LPS-205450

When generating the search iterator url, the searchCategoryId was not being appended and the pagination was not working correctly.

### CODE REVIEW

**Comment by SamZiemer** [comment]
ci:test:relevant

File: modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/view.jsp [hunk 1]
```diff
@@ -50,6 +52,10 @@ if (cur2 > 0) {
 
 portletURL.setParameter("mbCategoryId", String.valueOf(categoryId));
```
[the remark this answers is not in the document]
Comment (SamZiemer) [reply modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/view.jsp:52]: Thanks @robertoDiaz, I've added those changes.

**Comment by SamZiemer** [comment]
ci:test:sf

**Comment by SamZiemer** [comment]
ci:reevaluate:1387401_10166

**Review by robertoDiaz — APPROVED** [review]

**Comment by robertoDiaz** [comment]
ci:forward

**Comment by SylviaLuan** [comment]
ci:test:stable

**Comment by SamZiemer** [comment]
ci:test:relevant

**Comment by SamZiemer** [comment]
@robertoDiaz,

It seems the issues this PR keeps failing with are not related, when is it safe to force forward PRs?

**Comment by robertoDiaz** [comment]
ci:forward:force

**Pull request closed without merging**